Makefile

```diff
@@ -2,7 +2,7 @@
 .PHONY: build test check clean fmt cdc kafka_consumer coverage \
 	integration_test_build integration_test integration_test_mysql integration_test_kafka bank \
 	dm dm-master dm-worker dmctl dm-syncer dm_coverage \
-	engine tiflow df-demo df-chaos-case
+	engine tiflow tiflow-demo tiflow-chaos-case
```

Comment:
Is sub-command more appropriate here? Such as `tiflow demo` and `tiflow chaos`.

Reply:
I prefer to use standalone binary, for two reasons
- demo and chaos test will not be used in product environment
- It will make binary too large to put them together.